src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -143,11 +149,11 @@ private MetricItem Collect(bool isDelta)
                 {
                     // Record all observable instruments
                     this.listener.RecordObservableInstruments();
+                    var dt = DateTimeOffset.UtcNow;
```

Comment:
unrelated change. This reads time once for entire Collect() for all the aggregators, instead of each aggregator reading time itself.

Reply:
This might be a problem for synchronous instruments. Not a blocker at this moment.